deploy-manage/security/k8s-transport-settings.md

```diff
@@ -30,7 +30,9 @@ spec:
 Check the [Kubernetes Publishing Services (ServiceTypes)](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) that are currently available.
 
 ::::{note}
-When you change the `clusterIP` setting of the service, ECK deletes and re-creates the service, as `clusterIP` is an immutable field. This will cause a short network disruption, but in most cases it should not affect existing connections as the transport module uses long-lived TCP connections.
+When you change the `clusterIP` setting of the service, ECK deletes and re-creates the service, as `clusterIP` is an immutable field. This causes a short network disruption, but it should not affect existing connections as the transport module uses long-lived TCP connections.
+
+If you change the service’s `port`, set `targetPort` to `9300`, which is the default {{es}} transport interface port. Otherwise, Kubernetes uses the same value for both, resulting in failed connections.
```

Comment:
consider adding bullets here, or making this stacked notes. when I read the first paragraph of a note, I make the assumption that the second paragraph is related to the first. one of these methods will help people to understand that these are two notes that are not related

Reply:
Interesting, @shainaraskas , thanks for the hint.
In this case both paragraphs are related in the way that it's a note related with customizing service fields. First paragraph warns the user about what happens if they change `clusterIP` and the second warns the user about implications of changing the `port`.

How would you present this instead? 2 separates notes don't feel necessary here. Maybe an intro sentence + 2 bullets?